src/python/tests/appengine/handlers/cron/cleanup_test.py

```diff
@@ -367,6 +367,30 @@ def test_mark_issue_as_closed_if_testcase_is_fixed_10(self):
     self.assertNotEqual(self.issue.status, 'Verified')
     self.assertEqual('', self.issue._monorail_issue.comment)
 
+  def test_mark_issue_as_closed_if_testcase_is_fixed_11(self):
+    """Ensure that we mark issue as verified, but don't close it if job
+    definition specifies to skip it."""
+    testcase = test_utils.create_generic_testcase()
+    testcase.bug_information = str(self.issue.id)
+    testcase.fixed = '1:2'
+    testcase.open = False
+    testcase.one_time_crasher_flag = False
+    testcase.put()
+
+    data_types.Job(
+        name=testcase.job_type,
+        platform='LINUX',
+        environment_string=('SKIP_AUTO_CLOSE_ISSUE = True\n')).put()
+
+    cleanup.mark_issue_as_closed_if_testcase_is_fixed(
+        policy=self.policy, testcase=testcase, issue=self.issue)
+    self.assertIn(
+        'ClusterFuzz testcase 1 is verified as fixed in '
+        'https://test-clusterfuzz.appspot.com/revisions'
+        '?job=test_content_shell_drt&range=1:2',
+        self.issue._monorail_issue.comment)
+    self.assertEqual(self.issue.status, 'Assigned')
```

Comment:
nit: reverse order of args

Reply:
yes can do in another cl, since order is incorrect for other status comparison.